CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+## Unreleased
+
+### Added
+- Migrate API translation code to plugin infra
+- Support for self transpilation with improved error reporting
```

Comment:
this is just a testing thing; has no outside benefit as self-transpilation renders junk.

Reply:
Will remove